.github/workflows/ci.yaml

```diff
@@ -182,13 +182,3 @@ jobs:
 
       - name: Run browser tests
         run: make browser-test
-
-  review-dependabot-pr:
-    permissions:
-      contents: write
-      pull-requests: write
-    runs-on: ubuntu-latest
-    if: ${{ github.actor == 'dependabot[bot]' }}
-    steps:
-      - name: Enable Dependabot automation
-        uses: mozilla/syseng-pod/actions/dependabot-automerge@main
```

Comment:
😢 
Since Dev dependencies are not "deployed", shall we keep this automerge for them?

Reply:
We set up a dependabot group for all development dependencies so while it's not automatic, it's still fewer clicks. 

I'm still overall wary about automatically merging code from a security perspective. Even running a compromised linting tool locally (for example) could pose a risk.